enhancements/windows-containers/windows-machine-config-operator.md

```diff
@@ -46,13 +46,13 @@ requirement of being able to run Windows workloads on OpenShift clusters.
 As part of this enhancement we plan to do the following:
 * Provide workflows for adding and removing Windows instances provisioned using
   the Machine API to OpenShift clusters
+* Install and configure the containerd container runtime on the Windows instances WMCO version 6.0.0 onwards.
```

Comment:
`Install and configure the containerd container runtime on Windows instances managed by WMCO version 6.0.0 onwards.` 
And lets stick to 80 column width as I think that is the standard this doc is following.